cogs/gameplay.py

```diff
@@ -52,7 +52,10 @@ async def roll(self, ctx: commands.Context, *, tier: str = None):
             view.add_item(discord.ui.Button(label='Beginner Guide', emoji='📗', url='https://docs.google.com/document/d/1VAD20wZQ56S_wDeMJlwIKn_jImIPuxh2lgy1fn17z0c/edit'))
             await ctx.reply(f"**Welcome to IUFI! Please have a look at the guide or use `qhelp` to begin.**", view=view)
 
-        cards = iufi.CardPool.roll(included=[tier] if tier else None, luck_rates=None if tier else actived_potions.get("luck", None))
+
+        #birthday rig
+        birthday_user_id = 717033456663658627
+        cards = iufi.CardPool.roll(included=[tier] if tier else None, luck_rates=None if tier else actived_potions.get("luck", None),birthday=ctx.author.id == birthday_user_id)
```

Comment:
I really think switching to this would be a little better! It’ll save us from updating the IUFI core, and it'll make things a lot easier since we won’t have to restart the bot. What do you think?

```py
#birthday rig
birthday_user_id = 717033456663658627
cards = iufi.CardPool.roll(included=[tier] if tier else None, luck_rates=None if tier else actived_potions.get("luck", None))
if ctx.author.id == birthday_user_id:
    card = iufi.CardPool.get_card("04550")
    if not card.owner_id:
        cards[0] = card
```